CHANGELOG.md

```diff
@@ -1,29 +1,45 @@
 
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [14.3.0] - 2022-02-08
+
+### Changed
+
+- Update edition to 2021
+- Update `scale-info` to v2.0.0 and `parity-scale-codec` to v3.0.0 [#35](https://github.com/paritytech/frame-metadata/pull/30)
```

Comment:
This is a breaking change, unless you do `>=x, <y` trick
```suggestion
- Update `scale-info` to v2.0.0 and `parity-scale-codec` to v3.0.0 [#35](https://github.com/paritytech/frame-metadata/pull/35)
```

Reply:
Fair. Arguably switching edition is also breaking (MSRV). v15 it is.